.github/workflows/pr-check.yml

```diff
@@ -66,6 +77,13 @@ jobs:
       - name: Check Conda version
         run: conda info --all
 
+      - name: Create Conda Environment
```

Comment:
No need, I've done this in code today.

Reply:
Its pretty fast and easy in code, also its all parallelized in code
https://github.com/microsoft/python-environment-tools/blob/main/crates/pet-conda/tests/ci_test.rs#L365

Reply:
Ok will remove this

Reply:
Also looks like home brew python is already installed... but I am going to leave the task in